go/teams/service_helper.go

```diff
@@ -260,6 +263,84 @@ func AddMember(ctx context.Context, g *libkb.GlobalContext, teamname, username s
 	return AddMemberByID(ctx, g, team.ID, username, role)
 }
 
+type AddMembersRes struct {
+	Invite   bool                     // Whether the membership addition was an invite.
+	Username libkb.NormalizedUsername // Resolved username. May be nil for social assertions.
+}
+
+// AddMembers adds a bunch of people to a team. Assertions can contain usernames or social assertions.
+// Always returns a list where len(res) = len(assertions) and in corresponding order.
+// If err->nil, then all additions succeeded.
+// Adds the happy path members in a transaction and then does the stragglers one by one.
+func AddMembers(ctx context.Context, g *libkb.GlobalContext, teamname string, assertions []string, role keybase1.TeamRole) (res []AddMembersRes, err error) {
+	tracer := g.CTimeTracer(ctx, "team.AddMembers", true)
+	defer tracer.Finish()
+	failedRes := make([]AddMembersRes, len(assertions))
+	teamName, err := keybase1.TeamNameFromString(teamname)
+	if err != nil {
+		return failedRes, err
+	}
+	teamID, err := ResolveNameToID(ctx, g, teamName)
+	if err != nil {
+		return failedRes, err
+	}
+
+	err = RetryOnSigOldSeqnoError(ctx, g, func(ctx context.Context, _ int) error {
+		res = make([]AddMembersRes, len(assertions))
+		team, err := GetForTeamManagementByTeamID(ctx, g, teamID, true /*needAdmin*/)
+		if err != nil {
+			return err
+		}
+
+		tx := CreateAddMemberTx(team)
+		type sweepEntry struct {
+			Assertion string
+			UV        keybase1.UserVersion
+		}
+		var sweep []sweepEntry
+		for i, assertion := range assertions {
+			username, uv, invite, err := tx.AddMemberByAssertion(ctx, assertion, role)
+			if err != nil {
+				if strings.Contains(err.Error(), "doesn't have a Keybase account yet") {
```

Comment:
since this error also comes from the service (and not api or anything), wouldn't you rather have this as separate error type?